src/tools/rest-frontend/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ else ()
     # install files to destination
     install (DIRECTORY public resources DESTINATION ${install_directory})
     install (FILES package.json README.md ${CMAKE_CURRENT_BINARY_DIR}/Gruntfile.js DESTINATION ${install_directory})
-    if (NOT EXISTS ${install_directory}/application-config.json)
+    if (NOT EXISTS ${CMAKE_INSTALL_PREFIX}/${install_directory}/application-config.json)
```

Comment:
> my approach (default config + custom config) seems more interesting tbh. Or to stay with the current approach, I could also execute a copy process during the installation that does not override.

Yes, default config + custom config (where custom does not get installed but gets mounted for modifications) is a good idea. Complications in the installation procedure are better to be avoided (there are some obstacle, some we can deal with, like "DESTDIR", but others we cannot deal with: the approach simply won't work when Elektra is installed as package).